front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/tag/TaggedEntityDAO.groovy

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.front50.model.tag
+
+import com.netflix.spinnaker.front50.model.ItemDAO
+
+interface TaggedEntityDAO extends ItemDAO<TaggedEntity> {
+  Set<TaggedEntity> all(String prefix)
```

Comment:
`prefix` is something along the lines of:

- `application/front50`
- `cluster/front50-main`

It will match any tagged entity with a id _starting with_ `prefix`